spec/syntax.md

```diff
@@ -452,6 +452,9 @@ Otherwise, the set of characters allowed in names is large.
 The _nmtoken_ token doesn't have _name_'s restriction on the first character
 and is used as variant keys and option values.
 
+The _argtoken_ is used for expression arguments,
+and matches _nmtoken_ except for not allowing `-` as a first character.
```

Comment:
As above, perhaps add text about why `argtoken` exists, e.g.:

> `argtoken` exists as a convenience, since it allows some literal values to be unquoted.

Reply:
Done.